platform-hub-web/src/app/kubernetes-tokens/kubernetes-tokens-form.component.js

```diff
@@ -30,6 +30,10 @@ function KubernetesTokensFormController($state, hubApiService, logger, _, Kubern
     ctrl.isNew = !cluster;
     ctrl.loading = true;
 
+    if (!FeatureFlags.isEnabled(FeatureFlags.keys.kubernetesTokens)) {
+      $state.go('home'); // feature disabled
+    }
```

Comment:
I have an idea on how to do this at the routing level – let's pair on this when you're back.